docs/blogs/EnhancedDevPreviewGatewayAPI/IntroEnhancedDevPreview.md

```diff
@@ -0,0 +1,45 @@
+# Gateway API with OpenShift Networking - Developer Preview Update
+
+Authors: Candace Holman and Grant Spence
+
+![Gateway API Dev Preview Update](images/gateway-api-dev-preview-update.png)
+
+As a follow-on to the Openshift 4.12 Developer Preview of Gateway API, OpenShift has released an enhanced Developer
```

Comment:
Just a suggestion—it feels more natural to me to say that Red Hat has done something involving OpenShift than to say that OpenShift has done something.  (Sometimes we talk about "the OpenShift org", but that's more of an internal thing.)
```suggestion
As a follow-on to the Openshift 4.12 Developer Preview of Gateway API, Red Hat has released an enhanced Developer
```

Reply:
Done.